modules/cpmso-yaml-provider-spec-nutanix.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-nutanix_{context}"]
+= Sample Nutanix provider specification
+
+When you create a control plane machine set for an existing cluster, the provider specification must match the `providerSpec` configuration in the control plane machine custom resource (CR) that is created by the installation program.
+
+[discrete]
+[id="cpmso-yaml-provider-spec-nutanix-oc_{context}"]
+== Values obtained by using the OpenShift CLI
+
+In the following example, you can obtain some of the values for your cluster by using the OpenShift CLI.
+
+Infrastructure ID:: The `<cluster_id>` string is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+.Sample Nutanix `providerSpec` values
+[source,yaml]
+----
+providerSpec:
+  value:
+    apiVersion: machine.openshift.io/v1
+    bootType: "" <1>
+    categories: <2>
+        key: <prism_category_name>
+        value: <prism_category_value>
+    cluster: <3>
+        type: uuid
+        uuid: <cluster_uuid>
+    credentialsSecret:
+        name: nutanix-credentials <4>
+    image: <5>
+        name: <cluster_id>-rhcos
+        type: name
+    kind: NutanixMachineProviderConfig <6>
+    memorySize: 16Gi <7>
+    metadata:
+        creationTimestamp: null
+    project: <8>
+        type: ""
+    subnets: <9>
+    - type: uuid
+        uuid: <subnet_uuid>
+    systemDiskSize: 120Gi <10>
+    userDataSecret:
+        name: master-user-data <11>
+    vcpuSockets: 8 <12>
+    vcpusPerSocket: 1 <13>
+----
+<1> Specifies the boot type that the control plane machines use. For more information about boot types, see link:https://portal.nutanix.com/page/documents/kbs/details?targetId=kA07V000000H3K9SAK[Understanding UEFI, Secure Boot, and TPM in the Virtualized Environment]. Valid values are `Legacy`, `SecureBoot`, or `UEFI`. The default is `Legacy`.
++
+[NOTE]
+====
+You must use the `Legacy` boot type in {product-title} {product-version}.
+====
+<2> Specifies a Nutanix Prism category to apply to control plane machines. This stanza requires `key` and `value` parameters for a category key-value pair that exists in Prism Central. For more information about categories, see link:https://portal.nutanix.com/page/documents/details?targetId=Prism-Central-Guide-vpc_2022_6:ssp-ssp-categories-manage-pc-c.html[Category management].
+<3> Specifies a cluster configuration. In this example, the cluster type is `uuid`, so there is a `uuid` stanza.
```

Comment:
it might be useful to spell out that the cluster here refers to the Nutanix Prism Element Cluster

Reply:
Yes, definitely - I would have assumed OCP cluster :slightly_smiling_face: Will fix that, otherwise do you think this is ready to move along?

Reply:
We used to include all that command information in the callouts, but it became difficult to parse for platforms that have multiple values to determine (not just `cluster_id`, as Nutanix has), so we have been moving it out of the callout lists going forward.